crates/vm/levm/src/precompiles.rs

```diff
@@ -805,7 +862,7 @@ fn blake2f_compress_f(
     Ok(output)
 }
 
-// Reads a part of the calldata and returns what is read as u64 or an error
+/// Reads a part of the calldata and returns what is read as u64 or an error
```

Comment:
```suggestion
/// Reads part of the calldata and returns what is read as u64 or an error
```
Maybe it would be good if we clarify in which scenario it returns an error. "or an error if out of bounds" for example